src/compute-model-statistics/src/test/scala/VerifyComputeModelStatistics.scala

```diff
@@ -47,17 +63,15 @@ class VerifyComputeModelStatistics extends TestBase {
 
     val evaluatedData = new ComputeModelStatistics().transform(datasetWithScores)
     val firstRow = evaluatedData.first()
-    assert(firstRow.get(0) == 0.0)
-    assert(firstRow.get(1) == 0.0)
-    assert(firstRow.get(2) == 1.0)
-    assert(firstRow.get(3) == 0.0)
+    assert(tolEq.areEquivalent(firstRow.get(0).asInstanceOf[Double], 0.0))
+    assert(tolEq.areEquivalent(firstRow.get(1).asInstanceOf[Double], 0.0))
+    assert(tolEq.areEquivalent(firstRow.get(2).asInstanceOf[Double], 1.0))
+    assert(tolEq.areEquivalent(firstRow.get(3).asInstanceOf[Double], 0.0))
 
```

Comment:
Is there a test for using CMS without the Schema manipulation API?

Reply:
yep, test("Verify computing statistics on generic spark ML estimators is supported") { 